x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/fielddata_frequency_filter_percentage.tsx

```diff
@@ -20,13 +21,23 @@ export const FielddataFrequencyFilterPercentage = ({ min, max }: Props) => {
   };
 
   return (
-    <EuiDualRange
-      min={0}
-      max={100}
-      value={[min.value as number, max.value as number]}
-      onChange={onFrequencyFilterChange}
-      showInput
+    <EuiFormRow
       fullWidth
-    />
+      label={
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.fielddata.frequencyFilterPercentageFieldLabel"
+          defaultMessage="Min/max frequency percentage"
+        />
+      }
+    >
+      <EuiDualRange
```

Comment:
This component has a similar screen-reader accessibility issue. I created https://github.com/elastic/eui/issues/2737 to track this in EUI.